Examples-Web/IdleTimeout/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="theme-color" content="#000000" />
+        <title>Touch-Free React Project</title>
```

Comment:
Similar to the package name - might be worth making this something more specific to the example